assets/src/components/NavigationType/NavigationType.js

```diff
@@ -10,12 +10,12 @@ const NAVIGATION_TYPE_MINIMAL = 'minimal';
 export const NavigationType = ({value, setValue, defaultValue, options}) => {
   const defaultOptions = [
     {
-      "label": __( 'Main website navigation', 'planet4-blocks-backend' ),
-      "value": NAVIGATION_TYPE_PLANET4,
+      'label': __( 'Main website navigation', 'planet4-blocks-backend' ),
```

Comment:
Can't we just remove the quotes entirely in this case? 🤔

Reply:
Made this changes and also rebased my branch to use the latest changes in main and node version too. I will also update some fixes in other branches I am currently working on.